orchestrate.py

```diff
@@ -69,13 +69,16 @@ def get(self, path=None):
         self.render("loading.html", path=path)
 
 
-class StatsHandler(BaseHandler):
+class APIStatsHandler(BaseHandler):
     def get(self):
         '''Returns some statistics/metadata about the tmpnb server'''
         response = {
                 'available': len(self.pool.available),
                 'capacity': self.pool.capacity,
-                'version': '0.0.1-dev',
+                'version': '0.1.0',
+                'container_image': self.pool.container_config.image,
+                'docker_version': self.spawner.docker_client._docker_client.version(),
+                'docker_containers': self.spawner.docker_client._docker_client.containers()
```

Comment:
This is going to be a long-blocking call that won't hold up well in quick calls for stats on e.g. try.jupyter.org. As much as I'd *want* to see it, this isn't practical in current Docker and Go versions.